### ISSUE TITLE
Get-Help does not display synopsis for some cmdlets

### ISSUE THREAD

**Issue by sdwheeler** [PowerShell/PowerShell#9208]
The problem starts in the markdown file. If there is a blank line after the H3 SYNOPSIS header then Get-Help will not display the synopsis text. 

The workaround is to remove the blank line (contrary to markdown style guidance). 

I don't know if this is a problem in Get-Help or in the way that PlatyPS creates the MAML.

**Comment by iSazonov** [comment]
@sdwheeler Could you please add repo steps so we can easy start to debug?

**Comment by sdwheeler** [comment]
This has mostly been fixed in PS6 by removing the blank line after the header. But the repro is easy. Add a blank line after the `### SYNOPSIS` header and then build the help file with PlatyPS.

**Comment by ThomasNieto** [comment]
@sdwheeler is this still an issue?

**Comment by sdwheeler** [comment]
Yes. This is a combination of issues between the PlatyPS schema and the Get-Help cmdlet.

**Comment by doctordns** [comment]
@sdwheeler - it's been 2 years nearly - is this issue still unresolved?

**Comment by sdwheeler** [comment]
This is a problem with both Platyps and Get-Help.

We are working on PlatyPS but no plans for Get-Help any time soon.
